tutorials/using-bucket-website-with-mkdocs/index.mdx

```diff
@@ -0,0 +1,380 @@
+content:
+  h1: Creating a documentation website using Object Storage bucket website and Material for MkDocs
+  paragraph: Create a sleek documentation website using the Scaleway Object Storage bucket website feature and Material for MkDocs. Simple, fast, and easy to complete!
+categories:
+  - object-storage
+tags: documentation-website bucket-website material-for-mkdocs object-storage website
+dates:
+  validation: 2024-10-07
+  posted: 2024-10-07
+---
+
+This tutorial shows you how to create a documentation website using the Scaleway [Object Storage bucket website](/storage/object/concepts/#bucket-website) feature with MkDocs.
+
+[MkDocs](https://www.mkdocs.org/) is a static site generator that allows users to generate a static website from a set of Markdown files. It is written in Python and uses Markdown for content creation.
+We will use MkDocs to generate our website, and Material for MkDocs which is a documentation framework that extends MkDocs with additional features and customization options, to create visually appealing and user-friendly documentation websites.
+
+Find out how to Set up MkDocs and configure your website in the first tab, then click the second tab to learn how to get a preview of your website and how to deploy it.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Installed Python
+- Installed pip
+- Installed Visual Studio Code
+- Knowledge of the [Markdown](https://www.markdownguide.org/getting-started/#what-is-markdown) markup language
+
+<Tabs id="install">
+  <TabsTab label="Set up MkDocs and configure your website">
+    1. Open the Visual Studio Code terminal and paste the following command to create a virtual environment to host your MkDocs content:
+        ```
+         python -m venv envname
+        ```
+    2. Activate the virtual environment:
+
+        ```
+         source envname/bin/activate
+        ```
+    3. Create a folder inside your virtual environment. This folder will contain your documentation directory.
+
+        ```
+         mkdir my-folder
+        ```
+        <Message type="note">
+          Avoid naming your folder `docs` as this is already the default name for the MkDocs directory.
+        </Message>
+    4. Install MkDocs and the MkDocs Material theme in your virtual environment:
+        ```
+         pip install mkdocs
+         pip install mkdocs-material
+        ```
+    5. Open the documentation folder you have created and enter the following command:
+        ```
+         mkdocs new .
+        ```
+        <Message type="note">
+          This will create a YAML file named `mkdocs.yaml`, and a directory named `docs` containing an `index.md` file inside `my-folder`:
```

Comment:
```suggestion
          This will create a YML file named `mkdocs.yml`, and a directory named `docs` containing an `index.md` file inside `my-folder`:
```